libs/shared/src/consts/providers/channels/sms.ts

```diff
@@ -153,4 +154,12 @@ export const smsProviders: IProviderConfig[] = [
     docReference: 'https://developers.termii.com/authentication',
     logoFileName: { light: 'termii.png', dark: 'termii.png' },
   },
+  {
+    id: SmsProviderIdEnum.Africastalking,
```

Comment:
Seeing the branded name I'd go for `AfricasTalking` / `africasTalking` capitalisation everywhere as needed.